src/Components/PresentationalComponents/Snippets/SnippetWithPopover.js

```diff
@@ -90,7 +91,7 @@ const SnippetWithPopover = props => {
             aria-label={'Business risk popover'} position="left" footerContent={footerContent}>
 
             <Button variant="link"
-                isDisabled={(systemStatusId === 0) ? true : false}
+                isDisabled={hasDefaultStatus ? true : false}
```

Comment:
since the `hasDefaultStatus` returns a boolean, you can simplify it by
`isDisabled={hasDefaultStatus}`